1.8/usage/jobs/index.md

```diff
@@ -5,7 +5,7 @@ feature_maturity: preview
 menu_order: 80 
 ---
 
-You can create scheduled jobs in DC/OS without installing a separate service. Create and administer jobs in the DC/OS web interface or via an API.
+You can create scheduled jobs in DC/OS without installing a separate service. Create and administer jobs in the DC/OS web interface, the DC/OS CLI, or via an API.
 
 **Note:** The Jobs functionality of DC/OS is provided by the [Metronome](https://github.com/dcos/metronome) component, an open source Mesos framework that comes pre-installed with DC/OS. You may sometimes see the Jobs functionality referred to as "Metronome" in the DC/OS web interface and in logs.
```

Comment:
I don't believe there is a reference to metronome in the DCOS web interface.   There are 2 places this is exposed to my understanding.  1) the service endpoint is `/service/metronome` and the logs on the master will be under metronome.